ui/00_message/achievement/achievement_name.xml

```diff
@@ -6268,14 +6295,16 @@
   <message>
     <key>ACHIEVEMENT_NAME_2546</key>
     <original>目覚めた収集家</original>
-    <translation>Collector Arisen</translation>
+    <translation>Arisen Collector</translation>
```

Comment:
I changed it to match with the other related achievements, as the others refer to the "achiever" as an adjective of some sort. I will change it to whatever you think is best. Everything else you have commented on is fixed.